packages/nodes-base/nodes/QuestDb/QuestDb.node.ts

```diff
@@ -107,22 +107,6 @@ export class QuestDb implements INodeType {
 				required: true,
 				description: 'Name of the table in which to insert data to.',
 			},
-			{
-				displayName: 'Columns',
-				name: 'columns',
-				type: 'string',
-				displayOptions: {
-					show: {
-						operation: [
-							'insert',
-						],
-					},
-				},
-				default: '',
-				placeholder: 'id,name,description',
-				description:
-					'Comma separated list of the properties which should used as columns for the new rows.',
-			},
```

Comment:
Does QuestDB not support column type casting?

Reply:
It kinda does, but actually this column list was not being used. The implementation gets the json information passed in and uses json keys as columns so I removed this option as it's useless.